chrome.js

```diff
@@ -106,6 +144,73 @@ class ChromeWebDriverInstaller extends WebDriverInstallerBase {
         archiveUrl, binaryName, outputName,
         outputDirectory, /* isZip= */ true);
   }
+
+  /**
+   * @param {number} majorVersion
+   * @param {string} driverVersion
+   * @param {string} outputDirectory
+   * @return {!Promise}
+   */
+  async installNew_(majorVersion, driverVersion, outputDirectory) {
+    let platform;
```

Comment:
Line 155-169 can be extracted to a `getPlatform` function

Reply:
No, the platform strings used in the old and new styles differ slightly.

Reply:
Ex: Before 115, it was mac_arm64 for Arm and mac64 Intel.  From 115 on, it's mac-arm64 and mac-x64.
Ex: Before 115, it was win32 for Windows.  From 115 on, it's win64.